src/ScottPlot4/ScottPlot/Plot/Plot.Add.cs

```diff
@@ -240,6 +240,16 @@ public BarSeries AddBarSeries(List<Bar> bars)
             return barSeries;
         }
 
+        /// <summary>
+        /// Create a BarSeries filled with the given bars, add it to the plot, and return it.
+        /// </summary>
+        public BarSeries AddBarSeries(Bar[] bars)
```

Comment:
I think we can merge the two into `AddBarSeries(IList<Bar> bars)`, as arrays implement `IList<T>`.

There is however a difference in behaviour between the two overloads, the `Bar[]` overload you've written copies into a list, and the existing `List<Bar>` implementation passes the list in directly, meaning mutating the list will be reflected in `BarSeries`.

I think we have to maintain that existing behaviour (probably with a type guard), as we can't change the type of `BarSeries.Bar` because that would be a breaking change. It's potentially confusing that passing in different implementers of `IList<T>` would have different behaviour, so maybe we should separate these into two separate functions with separate names.